engine/src/main/java/org/destinationsol/game/projectile/ProjectileConfigs.java

```diff
@@ -50,7 +50,7 @@ public ProjectileConfigs(OggSoundManager soundManager, EffectTypes effectTypes,
                 String texName = node.getString("tex");
                 TextureAtlas.AtlasRegion tex = Assets.getAtlasRegion(texName + "Projectile");
                 float texSz = node.getFloat("texSz");
-                float speed = node.getFloat("spdLen");
+                float speed = node.getFloat("speed");
```

Comment:
This will require PRs in all the modules out there... Better get this merged before GCI!

Reply:
I can take a look at other modules as well...

Reply:
@Cervator if I was granted team membership in https://github.com/orgs/DestinationSol/teams/module-push-team, I could probably push hotfixes for this as soon as this gets merged (Would be quite annoying to fork 12 repos just to change two lines in each)